### ISSUE TITLE
Forecasting in the renewal eqn session

### ISSUE THREAD

**Issue by kathsherratt** [nfidd/nfidd#127]
I think my view is that we shouldn't include the forecasting piece in the random walk model in the renewal equation session (introduced in #112). 

For me it interrupts the nice flow of building up the model and comparing line by line - especially by adding the `transformed data` and `generated quantities` blocks which aren't introduced at this point, unlike other blocks previously. 

Not the biggest deal but in an already busy session it kinda just raises questions that aren't answered. So I don't quite see the advantage, compared to adding forecasting into the stan code at the time that it's first used (I assume in [forecasting](https://github.com/nfidd/nfidd/pull/126/files#r1645219410)).

I know it means adding another stan model to the lineup but this doesn't really affect the learner. Is this a big barrier? I'm open to understanding more the motivation for introducing it earlier.

**Comment by sbfnk** [comment]
I agree that for didactic purposes I think it's best if models are the minimum needed at any stage (and build upon each other), even if at the expense of maintainability. The main downside I think is that if we ever want to change something in one model we then have to change it in all for consistency.

**Comment by sbfnk** [comment]
will then need to update data-raw/generate-example-forecasts.r once https://github.com/nfidd/nfidd/pull/126/files is merged

**Comment by seabbs** [comment]
I'm not sure totally where we are on this but I think it might have snuck back in due to someones dubious ability to rebase

**Comment by sbfnk** [comment]
no it's never made it due to someone's dubious practice of side-addressing issues in PRs centered around other ones #132 

We'll need a separate PR for this I think.
